Guesser/FilterTypeGuesser.php

```diff
@@ -65,7 +66,7 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
 
         switch ($metadata->getTypeOfField($propertyName)) {
             case 'boolean':
-                $options['field_type']    = 'sonata_type_boolean';
+                $options['field_type']    = BooleanType::class;
```

Comment:
We use an if statement that detect the current symfony form version, using the presence of the `getBlockPrefix` method.